src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,66 @@
+.menu {
+  background-color: $colorBlue;
+  box-sizing: border-box;
+  height: 100vh;
+  padding: $headerTopPadding 0;
```

Comment:
Checklist:
14. Fix menu for small screens (if there is not enough space for all the menu items)

Reply:
done